blocks/count-cards/count-cards.js

```diff
@@ -0,0 +1,48 @@
+import { createOptimizedPicture } from '../../scripts/aem.js';
+
+export default function decorate(block) {
+  /* change to ul, li */
+  const ul = document.createElement('ul');
+  [...block.children].forEach((row) => {
+    const li = document.createElement('li');
+    while (row.firstElementChild) li.append(row.firstElementChild);
+    [...li.children].forEach((div) => {
+      if (div.children.length === 1 && div.querySelector('img')) {
+        div.classList.add(...['count-card-image']);
+        div.getElementsByTagName('img')[0].classList.add(...['icon-svg-ethos', 'lazyloaded']);
```

Comment:
`add('icon-svg-ethos', 'lazyloaded')`. No need to spread arrays when we can add the arguments directly.

If we would have had dynamic arrays, then maybe it could have made sense.

Reply:
updated the code to not use arrays.